CHANGES.rst

```diff
@@ -202,6 +202,10 @@ astropy.io.misc
 astropy.io.fits
 ^^^^^^^^^^^^^^^
 
+- Create magic methods ``__copy__`` and ``__deepcopy__`` on an ``HDUList``
```

Comment:
Normally this is called "added support for `copy.copy` and `copy.deepcopy` for `HDUList`". No need to mention the magic method names in the changelog.